environments.json

```diff
@@ -1874,8 +1961,10 @@
     "platformtype": "engine",
     "note_id": "harmony-flag",
     "equals": "chrome60",
-    "obsolete": false,
+    "release": "2017-08-08",
+    "unstable": true,
```

Comment:
Node 8 is not "unstable"; every release is stable in semver, always. It's just not LTS. That's why they call it "current".